packages/handlersjs-http/lib/util/clean-headers.ts

```diff
@@ -1,7 +1,12 @@
+import { getLogger } from '@digita-ai/handlersjs-logging';
+
+const logger = getLogger();
 export const cleanHeaders = (headers: { [key: string]: string }): { [key: string]: string } =>
   Object.keys(headers).reduce<{ [key: string]: string }>(
     (acc, key) => {
 
+      logger.info('Cleaning headers', { key, value: headers[key] });
+
```

Comment:
This is going to add a sh ton of logs. I would leave this out. Headers will be logged in other places in `context` anyways.